### PR TITLE
addeed the replication repoorts datasource

### PR DESCRIPTION
**Pull request by AnikaAgiwal2711**
# Description
Replication Reports datasource to fetch the details of the replication job status

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Feature Pull Request

##### RESOURCE OR DATASOURCE NAME
replication_report

##### OUTPUT

AT passed 
image

UT passed 
image

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [X] I have maintained at least 80% code coverage
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Unit tests
- [X] Acceptance tests
coverage report 
image

### CODE REVIEW

**Comment by shenda1** [comment]
run e2e test ReplicationReportsDataSource

**Comment by AnikaAgiwal2711** [comment]
run e2e test ReplicationReportsDataSource

**Review by rounak-adhikary — APPROVED** [review]

File: powerscale/provider/replication_reports_datasource.go [hunk 1]
```diff
@@ -0,0 +1,954 @@
...
+							Description:         "The number of WORM committed files which needed to be reverted. Since WORM committed files cannot be reverted, this is the number of files that were preserved in the compliance store.",
+							MarkdownDescription: "The number of WORM committed files which needed to be reverted. Since WORM committed files cannot be reverted, this is the number of files that were preserved in the compliance store.",
+						},
+						"user_conflict_files_skipped": schema.Int64Attribute{
+							Computed:            true,
+							Description:         "The number of files with user conflicts skipped by this job.",
+							MarkdownDescription: "The number of files with user conflicts skipped by this job.",
+						},
+						// 	// "service_report": schema.ListNestedAttribute{
+						// 	// 	Computed:            true,
```
Comment (shenda1) [inline powerscale/provider/replication_reports_datasource.go:463]: Why its commented?

**Comment by Krishnan-Priyanshu** [comment]
run e2e test ReplicationReportsDataSource

**Review by Krishnan-Priyanshu — APPROVED** [review]

**Comment by AnikaAgiwal2711** [comment]
run e2e test ReplicationReportsDataSource

**Review by Krunal-Thakkar — APPROVED** [review]

**Pull request merged**